content/docs/api/json-rpc.mdx

```diff
@@ -0,0 +1,761 @@
+2. `Boolean` - If `true` it returns the full transaction objects, if `false` only the hashes of the transactions.
+
+**Returns**
+
+`Object` - A block object, or `null` when no block was found:
+
+- `number` - the block number. `null` when its pending block.
+- `hash` - hash of the block. `null` when its pending block.
+- `parentHash` - hash of the parent block.
+- `nonce` - hash of the generated proof-of-work. `null` when its pending block.
+- `sha3Uncles` - SHA3 of the uncles data in the block.
+- `logsBloom` - the bloom filter for the logs of the block. `null` when its pending block.
+- `transactionsRoot` - the root of the transaction trie of the block.
+- `stateRoot` - the root of the final state trie of the block.
+- `receiptsRoot` - the root of the receipts trie of the block.
+- `miner` - the address of the beneficiary to whom the mining rewards were given.
+- `difficulty` - integer of the difficulty for this block.
+- `totalDifficulty` - integer of the total difficulty of the chain until this block.
+- `extraData` - the "extra data" field of this block.
+- `size` - integer the size of this block in bytes.
+- `gasLimit` - the maximum gas allowed in this block.
+- `gasUsed` - the total used gas by all transactions in this block.
+- `timestamp` - the unix timestamp for when the block was collated.
+- `transactions` - Array of transaction objects, or 32 Bytes transaction hashes depending on the last given parameter.
+- `uncles` - Array of uncle hashes.
+
+**Example**
+
+```bash
+curl https://your-rpc-endpoint \
+-X POST \
+-H "Content-Type: application/json" \
+--data '{"method":"eth_getBlockByNumber", "params":["latest",false], "id":1,"jsonrpc":"2.0"}'
+```
+
+## eth_getTransactionByHash
+
+Returns the information about a transaction requested by transaction hash.
+
+**Parameters**
+
+1. `hash` - The hash of a transaction
+
+**Returns**
+
+`Object` - A transaction object, or `null` when no transaction was found:
+
+- `blockHash` - hash of the block where this transaction was in. `null` when its pending.
+- `blockNumber` - block number where this transaction was in. `null` when its pending.
+- `from` - address of the sender.
+- `gas` - gas provided by the sender.
+- `gasPrice` - gas price provided by the sender in Wei.
+- `hash` - hash of the transaction.
+- `input` - the data send along with the transaction.
+- `nonce` - the number of transactions made by the sender prior to this one.
+- `to` - address of the receiver. `null` when its a contract creation transaction.
+- `transactionIndex` - integer of the transactions index position in the block. `null` when its pending.
+- `value` - value transferred in Wei.
+- `v` - ECDSA recovery id
+- `r` - ECDSA signature r
+- `s` - ECDSA signature s
+
+**Example**
+
+```bash
+curl https://your-rpc-endpoint \
+  -X POST \
+  -H "Content-Type: application/json" \
+  --data '{"method":"eth_getTransactionByHash","params":["hash"],"id":1,"jsonrpc":"2.0"}'
+```
+
+## eth_getTransactionByBlockHashAndIndex
```

Comment:
This may not be relevant as I beleive in Shardeum all transactions have the same index in a block